test/api_getProduct_test.dart

```diff
@@ -1615,7 +1615,7 @@ void main() {
         barcode,
         language: OpenFoodFactsLanguage.GERMAN,
         fields: [ProductField.INGREDIENTS],
-        version: ProductQueryVersion.V0,
+        version: ProductQueryVersion.v0(),
```

Comment:
Don't know about you but for me this syntax looks a bit odd, I'd prefer a get here so that it's just v0 or v2